### PR TITLE
[3680] Fix Text Shifted Up On Mobile

### PR DESCRIPTION
**Pull request by mahmoud-Elshahawy**
Related issue(s):
* Fixes https://github.com/scandipwa/scandipwa/issues/3680

Problem:
* Mobile: Text is shifted up in checkboxes on PDP.

In this PR:
* I have added margin: auto; to ProductDownloadableLink-SampleTitle class in ProductDownloadableLinks.style.scss file.

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
**On Android:** Text is slightly shifted up in checkboxes 
Screenshot 2021-11-24 at 08 53 25

**On iOS:** Text is slightly shifted down in checkboxes 


**Expected result:**
Text is central aligned

**Comment by aleksandrakorolova** [comment]
Still, can reproduce
Screenshot 2021-11-25 at 09 58 04
Screenshot 2021-11-25 at 10 03 58

File: packages/scandipwa/src/component/ProductDownloadableSamples/ProductDownloadableSamples.style.scss [hunk 1]
```diff
@@ -17,7 +17,6 @@
     &-Title {
         font-weight: bold;
         margin-block-end: 5px;
-
```
Comment (arturs-pesockis) [inline packages/scandipwa/src/component/ProductDownloadableSamples/ProductDownloadableSamples.style.scss:20]: Blank line shouldn't be removed. Usually there should be separation between properties and `@includes`

File: packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.style.scss [hunk 2]
```diff
@@ -73,6 +73,8 @@
 
     &-SampleTitle {
         flex: 1;
+        // removed top
```
Comment (arturs-pesockis) [inline packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.style.scss:76]: comment is not needed here

**Review by arturs-pesockis — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/ProductDownloadableSamples/ProductDownloadableSamples.style.scss [hunk 3]
```diff
@@ -17,7 +17,7 @@
...
+
```
Comment (arturs-pesockis) [inline packages/scandipwa/src/component/ProductDownloadableSamples/ProductDownloadableSamples.style.scss:20]: remove trailing spaces

**Review by arturs-pesockis — CHANGES_REQUESTED** [review]

**Comment by aleksandrakorolova** [comment]
Still, Text is slightly shifted like as arrow in #3734. => So there is no proper way to avoid this issue on devices if Muli is used.

File: packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.style.scss [hunk 4]
```diff
@@ -73,6 +73,7 @@
 
     &-SampleTitle {
         flex: 1;
+        margin:auto;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/ProductDownloadableLinks/ProductDownloadableLinks.style.scss:76]: please space between property and value

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**